src/server/saved_objects/client/lib/create_find_query.js

```diff
@@ -35,5 +40,20 @@ export function createFindQuery(options = {}) {
     });
   }
 
-  return { version: true, query: { bool } };
+  const query = { version: true, query: { bool } };
+
+  if (sortField) {
+    const value = {
+      order: sortOrder,
+      unmapped_type: get(mappings, `${type}.properties.${sortField}.type`)
```

Comment:
Is there any chance that `sortField` or `type` could have a `.` in the value? If so, then the second parameter of lodash get should be an array like `[type, 'properties', sortField, 'type']`